lib/bojangles.rb

```diff
@@ -34,7 +35,11 @@ def prepare
       Departure.import GTFS::Data.stop_time_records
       GTFS::Files.mark_import_done
     end
-    client = GitHub::Client.new token: GITHUB_TOKEN
+    options = {}.tap do |opts|
```

Comment:
I've never actually realized how tap could be used. I thought "who would want that" and now I kind of see.